### ISSUE TITLE
Collaborating on CIMIS data 

### ISSUE THREAD

**Issue by patwater** [codeswitching/rcimis#2]
@dmarulli @christophertull check this out! 

Note @codeswitching Chris has been working on a spatial CIMIS parser which may be of interest

**Comment by christophertull** [comment]
Yeah I have been working on parsing Spatial CIMIS files directly from the FTP site. The CIMIS API is great when you are concerned with a particular location, but for our purposes we were more interested in having the ETo grid across the state.

Right now the parser is in `python` but no reason it couldn't be adapted to R. Expected usage for us would be 

1. select a date range
2. get all Spatial CIMIS data for that date range, transformed from a grid file (.asc) into a tidy dataframe indexed by date and grid coordinate.
